packages/core/src/performers/step-performer/StepPerformerPromptCreator.ts

```diff
@@ -258,17 +259,9 @@ export class StepPerformerPromptCreator {
     if (previousStep.result && !previousStep.error)
       return `- Result: ${previousStep.result}`;
     if (isMostPreviousStep && previousStep.error) {
-      const truncatedError = this.truncateString(previousStep.error, 2000);
+      const truncatedError = truncateString(previousStep.error, 2000);
```

Comment:
let's include the `limit` of `2000` as a default value in the `truncateString` method, it will be easier to configure it from there.